android/app/build.gradle

```diff
@@ -80,6 +80,9 @@ dependencies {
     implementation 'com.github.pedroSG94:RTSP-Server:1.0.8'
     implementation 'com.github.pedroSG94.rtmp-rtsp-stream-client-java:rtplibrary:2.0.2'
 
+    //RTP Client
+    implementation 'com.github.ar-android:libstreaming:1.0.0'
```

Comment:
We should use the original library.
https://github.com/fyhertz/libstreaming

Reply:
I tried the original library in the beginning. However, the official libstreaming published on jitpack is the very old version, and they didn't publish their latest version. I'm not sure if there is another way around. Do you have any idea?